### PR TITLE
Adding nginx combined log format

### PR DESCRIPTION
**Pull request by ebanDev**

### CODE REVIEW

**Comment by ebanDev** [comment]
See #13 ;)

File: README.md [hunk 1]
```diff
@@ -59,6 +59,16 @@ format transform `{request>remote_addr} - {request>user_id} [{ts}] "{request>met
 }
 ```
 
+## Nginx Combined Log Format Example
+
+The more spelled out way of doing it is:
+
+```caddy
+format formatted `{request>remote_addr} - {request>user_id} [{ts}] "{request>method} {request>uri} {request>proto}" {status} {size} "{request>headers>Referer>[0]}" "{request>headers>User-Agent>[0]}"` {
```
Comment (mholt) [inline README.md:67]:
```suggestion
format transform `{request>remote_addr} - {request>user_id} [{ts}] "{request>method} {request>uri} {request>proto}" {status} {size} "{request>headers>Referer>[0]}" "{request>headers>User-Agent>[0]}"` {
```

File: README.md [hunk 2]
```diff
@@ -59,6 +59,16 @@ format transform `{request>remote_addr} - {request>user_id} [{ts}] "{request>met
 }
 ```
 
+## Nginx Combined Log Format Example
```
Comment (mohammed90) [inline README.md:62]:
```suggestion
## Apache Combined Log Format Example
```

**Review by mohammed90 — CHANGES_REQUESTED** [review]

**Review by mohammed90 — CHANGES_REQUESTED** [review]
Thank you for adding another example of a common pattern! Since we're expanding into 2 examples now, can you please change the header into "Examples" with 2 sections, one for each example?

**Comment by ebanDev** [comment]
Should be good :) (sry for the terrible naming of my commits x) ig you should squash them)

**Review by mohammed90 — APPROVED** [review]
Thank you so much for the addition! It will help more folks looking for that particularly common format.

**Comment by ebanDev** [comment]
Pleasure ! :D

**Comment by ebanDev** [comment]
Pleasure ! :D

**Pull request merged**